crates/storage/src/cache.rs

```diff
@@ -85,13 +85,15 @@ impl<K: Clone + Eq + Hash, V: Clone> CacheTable<K, V> {
 
     /// Gets the number of elements in the cache.
     // TODO replace this with an atomic we update after every op
+    #[allow(dead_code)] // FIXME: remove this
```

Comment:
Same for this stuff, removed.

Reply:
Ok, once we have it up we can link the PRs by adding the number here?